.github/workflows/ci-hardening.yml

```
@@ -0,0 +1,158 @@
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Discover Dockerfiles
+        id: df
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:52: Double quote to prevent globbing and word splitting

(shellcheck)